server/app/views/BaseHtmlLayout.java

```diff
@@ -125,8 +125,9 @@ public HtmlBundle getBundle(HtmlBundle bundle) {
 
     // Add default stylesheets.
     bundle.addStylesheets(viewUtils.makeLocalCssTag(USWDS_STYLESHEET_FILEPATH));
+    // TODO: @Don, remove this. Just an example of using it.
+    // bundle.addStylesheets(viewUtils.makeLocalCssTag("dist/uswds_northstar.min"));
```

Comment:
I didn't feel like constantly flipping the setting. Comment out 127 and uncomment 129 and the main app will use the new northstar stuff now.

You'll want to remove this example, but `dist/uswds_northstar.min` is what you'll want to use when making the link for the css file. It probably needs to be `.min.css`, I'm guessing the viewutils add that.